packages/web-components/src/components/masthead/left-nav.ts

```diff
@@ -135,10 +135,12 @@ class DDSLeftNav extends StableSelectorMixin(BXSideNav) {
       const { expanded, _startSentinelNode: startSentinelNode, _endSentinelNode: endSentinelNode } = this;
       if (expanded) {
         this._hFocusWrap = focuswrap(this.shadowRoot!, [startSentinelNode, endSentinelNode]);
-        doc?.body?.classList.add(`${prefix}--body__lock-scroll`);
+        doc.body.style.position = `relative`;
+        doc.body.style.overflow = `hidden`;
       } else {
         const { selectorMenuSections, selectorFirstMenuSection } = this.constructor as typeof DDSLeftNav;
-        doc?.body?.classList.remove(`${prefix}--body__lock-scroll`);
+        doc.body.style.position = `inherit`;
+        doc.body.style.overflow = `auto`;
```

Comment:
hey @proeung further testing, I don't think we need to set the `position`, `overflow` should handle it!